### ISSUE TITLE
Build Universal app shell fails with error.

### ISSUE THREAD

**Issue by zbream** [zbream/oss-tracker#3]
Running `npm run build:shell` fails with the following error:

```
Unhandled Promise rejection: package.json does not exist at C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\package.json ; Zone: <root> ; Task: Promise.then ; Value: Error: package.json does not exist at C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\package.json
    at Object.PsoT.exports.find (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:1910809)
    at Object.wPNL (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:4005185)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.XpdW (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:2108241)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.g1pB (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:2855730)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.Ou8q (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:1873202)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.BYZf (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:1138880)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.IM/u (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:1761100)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.uWU0 (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:3959721)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.dZe3 (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:2298917) Error: package.json does not exist at C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\package.json
    at Object.PsoT.exports.find (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:1910809)
    at Object.wPNL (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:4005185)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.XpdW (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:2108241)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.g1pB (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:2855730)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.Ou8q (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:1873202)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.BYZf (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:1138880)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.IM/u (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:1761100)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.uWU0 (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:3959721)
    at __webpack_require__ (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:295)
    at Object.dZe3 (C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\server\main.js:1:2298917)
package.json does not exist at C:\Users\Zack\Source\_personal\oss-tracker\dist\oss-tracker\package.json
```

Likely caused by `rxfire`:

https://github.com/firebase/firebase-js-sdk/issues/2010

https://github.com/angular/angularfire/issues/2320#issuecomment-584448075

**Comment by limes** [comment]
+1

**Comment by zbream** [comment]
Closed via 5892af218876829570bda4d02cac7919f23ff735.
